style.css

```diff
@@ -0,0 +1,25 @@
+@media (max-width: 650px) {
+  * {
+    margin: 0 auto;
+  }
+  body {
+    max-width: 500px;
+    background-color: aqua;
+    font-family: "Lato", sans-serif;
+  }
+
+  header {
+    position: fixed;
+    top: 0;
+    min-height: 75px;
+    padding: 0px 20px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #eee;
+  }
+  input[type="submit"]:hover {
+    background-color: orange;
+    transition: background-color 1s;
+  }
+}
```

Comment:
- Please make an Indentation of two spaces. This will help your code to be clean.